### PR TITLE
Gaze real world testing

### PR DESCRIPTION
**Pull request by yvsriram**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist:

- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] I ran `tests/hw_manual_test.py` on hardware and observed that robot behavior looks normal.

### CODE REVIEW

File: projects/stretch_ovmm/requirements.txt [hunk 1]
```diff
@@ -17,4 +17,4 @@ fasttext
 scikit-learn
 lvis
 nltk
-git+https://github.com/openai/CLIP.git
\ No newline at end of file
+git+https://github.com/openai/CLIP.git
```
Comment (cpaxton) [inline projects/stretch_ovmm/requirements.txt:20]: i think we should move these into the main environment.yaml files too

File: src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 import numpy as np
 import torch
 from habitat.core.agent import Agent
+from habitat.core.spaces import EmptySpace
```
Comment (cpaxton) [inline src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py:18]: if we are adding habitat as a dependency we should update install insturctions accordingly

File: src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py [hunk 3]
```diff
@@ -216,7 +274,7 @@ def convert_to_habitat_obs_space(
             }
         )
 
-    def act(self, observations: Observations) -> Dict[str, int]:
+    def act(self, observations: Observations):
```
Comment (cpaxton) [inline src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py:277]: why this change?

File: src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py [hunk 4]
```diff
@@ -272,6 +334,15 @@ def act(self, observations: Observations) -> Dict[str, int]:
         )
 
 
+def convert_to_robot_action(cont_action):
```
Comment (cpaxton) [inline src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py:337]: what does do? can we add some more documentation?

**Review by cpaxton — APPROVED** [review]
minor comments but mostly lgtm

File: src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py (hunk 2 above)
Comment (yvsriram) [reply src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py:18]: added to `projects/stretch_ovmm/README.md`

File: src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py (hunk 4 above)
Comment (yvsriram) [reply src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py]: converts policy output habitat actions to home-robot actions. Added.

File: projects/stretch_ovmm/requirements.txt (hunk 1 above)
Comment (yvsriram) [reply projects/stretch_ovmm/requirements.txt:20]: The main environment.yaml already has openai-clip. @ykarmesh do you remember why this was required here?

Comment (cpaxton) [reply projects/stretch_ovmm/requirements.txt:20]: we should remove this?

**Review by cpaxton — APPROVED** [review]
lgtm
